examples/kind-multinode/quickstart-kind.sh

```
@@ -0,0 +1,121 @@
+      SIDELOAD_IMAGES=true
+      shift 1
+      ;;
+    --server-count-override)
+      SERVER_COUNT_OVERRIDE=$2
+      shift 2
+      ;;
+    --)
+      shift
+      break
+      ;;
+    *|-h|--help)
```

Shellcheck (0.11.0)

[warning] This pattern always overrides a later one on line 62.

(SC2221)